frontend/src/components/Navigation/HeaderNav.tsx

```diff
@@ -28,8 +28,6 @@ const menuItems: menuObject[] = [
  */
```

Comment:
we can delete the comments just above this line since you fixed the issue they mention :tada:

Reply:
Ah you're right, yay!